src/util/util.js

```diff
@@ -4,6 +4,7 @@
 const UnitBezier = require('@mapbox/unitbezier');
 const Coordinate = require('../geo/coordinate');
 const Point = require('point-geometry');
+const LngLat = require('../geo/lng_lat');
```

Comment:
This creates a circular dependency between util/util.js and geo/lng_lat.js (which uses util.wrap). Which seems to be causing an issue. Investigating...